QuizApp/App/Extensions/UIKit/UILabel+Extensions.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+extension UILabel {
+
+    func style(with text: String?, color: UIColor, alignment: NSTextAlignment, font: UIFont) {
```

Comment:
```suggestion
    func style(with text: String? = nil, color: UIColor, alignment: NSTextAlignment, font: UIFont) {
```
Add a default value for `text` parameter so you don't have to set nil explicitly.